core/services/assets/BlockAssetManager.php

```diff
@@ -110,6 +120,9 @@ public function getStyleHandle()
      */
     public function setStyleHandle($style_handle)
     {
+        if (strpos($style_handle, BlockInterface::NAME_SPACE . '-') !== 0) {
+            $style_handle = BlockInterface::NAME_SPACE . '-' . $style_handle;
+        }
```

Comment:
Regarding using `BlockInterface::NAME_SPACE` as the prefix for all these methods, will EE add-ons be able to override this to add their own prefix?  So instead of `eventespresso-script-handle`, an addon might have `eea-automated-upcoming-events-script-handle`?

Reply:
No the namespacing is just to ensure that our scripts don't conflict with another vendor. 
Add-ons will need to still supply a handle that is unique within the `eventespresso` namespace.
If an add-on wants to do that by prepending their own namespace, then that's fine with me, doesn't really matter.
So as long as the Automated Upcoming Events Notifications add-on (or any other) doesn't try to register a script with the handle `core-blocks` or any other handle that is already used, then everything will be fine.